site/src/components/Logs/LogLine.tsx

```diff
@@ -20,62 +19,51 @@ type LogLineProps = {
 export const LogLine: FC<LogLineProps> = ({ level, ...divProps }) => {
 	return (
 		<pre
-			css={styles.line}
-			className={`${level} ${divProps.className} logs-line`}
+			css={(theme) => ({
```

Comment:
what's going on here? why just inline these without converting?

Reply:
leaving it in the `styles` constant just makes the diff smaller and more focused, which is important for stuff like this where the things likely to go wrong are minutia. any extra noise makes it a lot easier to miss things.